sbor-derive-common/src/describe.rs

```diff
@@ -126,6 +128,27 @@ fn handle_normal_describe(
     let (impl_generics, ty_generics, where_clause, child_types, custom_type_kind_generic) =
         build_describe_generics(&generics, &attrs, context_custom_type_kind)?;
 
+    let type_name =
+        get_sbor_attribute_string_value(&attrs, "type_name")?.unwrap_or(ident.to_string());
+
+    let type_id = quote! {
+        ::sbor::RustTypeId::novel_with_code(
```

Comment:
Note for the future - we can consider just using `core::any::TypeId::of::<Self>` here, if we add where `Self: 'static` to the Describe trait (which should be fine) https://github.com/rust-lang/rust/pull/72488